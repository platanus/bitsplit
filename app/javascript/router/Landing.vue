<template>
  <div>
    <p>Route: {{ routeName }}</p>

    <router-link :to="signInRoute">sign in</router-link>
    <router-link :to="signUpRoute">sign up</router-link>
    <router-link :to="homeRoute">Home</router-link>
  </div>
</template>

<script>
import { mapState, mapGetters } from 'vuex'
export default {
  name: 'Landing',
  data() {
    return {
      routeName: 'Landing',
      signInRoute: 'sign-in',
      signUpRoute: 'sign-up',
      homeRoute: 'home'
    }
  }
}
</script>

<style lang="scss"></style>