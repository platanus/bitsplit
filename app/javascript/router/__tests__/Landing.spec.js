import { shallowMount } from '@vue/test-utils';
import Landing from '../Landing';

describe('Landing', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(Landing);
    expect(wrapper).toMatchSnapshot();
  });
});
