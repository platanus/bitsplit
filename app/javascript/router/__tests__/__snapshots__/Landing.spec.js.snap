// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing matches snapshot 1`] = `
<div class="m-12">
  <div class="bg-gray-200 p-10 my-4 rounded-md">
    <p class="text-5xl font-bold">BitSplit</p>
    <p class="font-light">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam</p>
  </div>
  <div class="flex flex-col justify-center md:flex-row md:justify-start bg-gray-200 p-10 rounded-md">
    <div class="flex flex-col md:flex-row md:items-start text-center px-4 py-2">
      <linkbutton-stub route="sign-in" classmod="bg-blue-500 hover:bg-blue-700 mx-4 my-3 md:my-0">Iniciar sesion</linkbutton-stub>
      <linkbutton-stub route="sign-up" classmod="bg-blue-500 hover:bg-blue-700 mx-4">Registrarse</linkbutton-stub>
    </div>
  </div>
</div>
`;
